src/lib/rest/rest.cpp

```diff
@@ -1410,7 +1410,22 @@ static int connectionTreat
   }
   else
   {
-    serveFunction(ciP);
+    if (metricsMgr.isOn())
+    {
+      struct timeval  start;
+      struct timeval  end;
+
+      gettimeofday(&start, NULL);
```

Comment:
We cannot go inside MHD but a good approach is at least to measure that time the transaction passes in CB code, i.e. the start should be the moment just after MHD call us and the end should be just before return to MHD.